domains/measurements/Measurements.API/Tests/SwaggerTests.GetSwaggerDocs_v1.0.verified.txt

```diff
@@ -0,0 +1,211 @@
+﻿{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Measurements API 1.0",
+    "description": "",
+    "version": "1.0"
+  },
+  "paths": {
+    "/api/measurements/consumption": {
```

Comment:
would it make sense to make one endpoint for all measurements and put a query param saying if it should be production or consumption?

These endpoints looks a lot like each other and i am guessing if we change something for consumption we should change it on production. So to avoid this i am suggestion making one endpoint

Reply:
This file is not used anymore after deletion of old endpoints